datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -194,7 +194,7 @@ export default function DefaultPreviewCard({
     }
     return (
         <PreviewContainer data-testid={dataTestID}>
-            <div>
+            <div style={{ width: '100%' }}>
```

Comment:
Minor: We try to avoid inline styling. Can you pull into a styled component plspls

Reply:
Actually - Can we remove this div altogether? And just put the styling on the parent "PreviewContainer" div?